pygfx/renderers/wgpu/shaders/meshshader.py

```diff
@@ -425,7 +425,14 @@ def _encode_morph_texture(self, geometry):
     def get_pipeline_info(self, wobject, shared):
         material = wobject.material
 
-        topology = wgpu.PrimitiveTopology.triangle_list
+        if isinstance(wobject, Line):
+            topology = wgpu.PrimitiveTopology.line_strip
+            # todo: wgpu.PrimitiveTopology.line_list, wgpu.PrimitiveTopology.line_loop
+        elif isinstance(wobject, Points):
+            topology = wgpu.PrimitiveTopology.point_list
+        else:
+            topology = wgpu.PrimitiveTopology.triangle_list
+            # todo: wgpu.PrimitiveTopology.triangle_strip
```

Comment:
> I assume this change is there because gltf files can contain lines and points, and to support these correctly we need to be able to put a mesh material on it?

Yes, that's correct. You can check this test case: https://github.com/KhronosGroup/glTF-Sample-Assets/blob/main/Models/PrimitiveModeNormalsTest/README.md

In glTF_viewer example:
![image](https://github.com/user-attachments/assets/ec31f4d2-9bff-400d-981c-c755880fe22a)